### ISSUE TITLE
Updated postgres script for calendar plugin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#350]
```
The current SVN does not contain an updated SQL script for database creation 
with postgresql (it fails with missing "url" column)

Please find the updated file.
I've removed the all_day_event since it was not in mysql.sql anymore, please 
correct this if I'm wrong.

BTW, the calendar plugin doesn't work with caldav either since it fails to find 
the "jscolor" plugin which is missing in the SVN repository.
I'm opening another issue for this one.


```

Original issue reported on code.google.com by `boite.po...@gmail.com` on 4 Jul 2012 at 4:53

Attachments:
- [postgres.sql](https://storage.googleapis.com/google-code-attachments/myroundcube/issue-350/comment-0/postgres.sql)

**Comment by GoogleCodeExporter** [comment]
```
Which SVN repository?

If you refer to myroundcube.googlecode.com, this repository is used internally 
only. To fetch recent code, you have to install plugin_manager.
```

Original comment by `rol...@liebl.ath.cx` on 5 Jul 2012 at 5:00
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in calendar 9.6.3. Please use plugin_manager to open tickets in future. 
Otherwise tickets will be rejected as invalid in future.
```

Original comment by `rol...@liebl.ath.cx` on 5 Jul 2012 at 5:06
- Changed state: **Fixed**
